odoo/addons/base/rng/tree_view.rng

```diff
@@ -20,6 +20,12 @@
         </rng:element>
     </rng:define>
 
+    <rng:define name="widget">
+        <rng:element name="widget">
+            <rng:attribute name="name"/>
+        </rng:element>
+    </rng:define>
```

Comment:
i'm not sure you need to define widget here, need to double check

Reply:
We can put it in the "common.rng" but for now only the tree view need this element.